sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/OffsetLimitQueryTests.java

```diff
@@ -8,6 +8,7 @@
 import com.azure.cosmos.CosmosAsyncContainer;
```

Comment:
nit: Unused imports. (not related to this PR)

Reply:
Fixed